suite-common/wallet-core/src/accounts/accountsReducer.ts

```diff
@@ -86,15 +87,35 @@ export const prepareAccountsReducer = createReducerWithExtraDeps(
 
 export const selectAccounts = (state: AccountsRootState) => state.wallet.accounts;
 
-export const selectAccount = (state: AccountsRootState, accountKey: string) =>
-    state.wallet.accounts.find(account => account.key === accountKey);
+export const selectAccount = createSelector(
+    selectAccounts,
+    (_state: AccountsRootState, accountKey: string) => accountKey,
+    (accounts, accountKey) => accounts.find(account => account.key === accountKey),
+);
+
+export const selectDeviceNetworkAccounts = createSelector(
```

Comment:
What about name here, this one seems little bit cryptic. According to new convetion it should be something like `selectAccountsByNetworkAndDevice`? It's quite long but in the other hand it super easy to read and tell what it is doing. What do you think?

Reply:
I like those super easy and readable names if there is a good reason and `selectAccountsByNetworkAndDevice` sounds much better than `selectDeviceNetworkAccounts` in this case. Fixed.